cmd/apikey/apikey_save.go

```diff
@@ -132,6 +133,27 @@ var apikeySaveCmd = &cobra.Command{
 			config.SaveConfig()
 		}
 
+		if config.Current.Clusters == nil {
+			client, err := civogo.NewClientWithURL(apiKey, config.Current.Meta.URL, config.Current.Meta.DefaultRegion)
+			if err != nil {
+				utility.Error("Unable to create a Civo API client, please report this at https://github.com/civo/cli")
+				os.Exit(1)
+			}
+			clusters, err := client.ListKubernetesClusters()
+			if err != nil {
+				fmt.Println(err.Error())
+				os.Exit(1)
+			}
+			config.Current.Clusters = make(map[string]bool)
+			for _, cluster := range clusters.Items {
+				config.Current.Clusters[cluster.ID] = true
+				if time.Since(cluster.BuiltAt) > (365 * 24 * time.Hour) {
```

Comment:
I think there's a time.Year, if not you can use time.Day *365 but more importantly, we have to check if it's divisible by the year in which cluster was built.